Userland/Libraries/LibC/regex.h

```diff
@@ -38,6 +38,7 @@ enum __Regex_Error {
     __Regex_InvalidCaptureGroup,        // Content of capture group is invalid.
     __Regex_InvalidNameForCaptureGroup, // Name of capture group is invalid.
     __Regex_InvalidNameForProperty,     // Name of property is invalid.
+    __Regex_DuplicateNamedCapture,      // Duplicate named capture group
```

Comment:
I only now realize that this is a LibC file, should all regex errors be added to this? All the other ones were so I just did it.

Reply:
Yeah, that's fine.